integrations/snyk/main.py

```diff
@@ -36,13 +36,15 @@ def generate_signature(payload: bytes, secret: str) -> str:
 
 
 def init_client() -> SnykClient:
+    config = ocean.integration_config
+    parse_ids = lambda key: [id_.strip() for id_ in config.get(key, "").split(",")] if config.get(key) else None
     return SnykClient(
-        ocean.integration_config["token"],
-        ocean.integration_config["api_url"],
-        ocean.integration_config.get("app_host"),
-        ocean.integration_config.get("organization_id"),
-        ocean.integration_config.get("groups"),
-        ocean.integration_config.get("webhook_secret"),
+        config["token"],
+        config["api_url"],
+        config.get("app_host"),
+        parse_ids("organization_id"),
+        parse_ids("groups"),
+        config.get("webhook_secret"),
```

Comment:
Lambda functions are good if the operation involved is a simple one line. But here I see it's not that straightforward and could make it hard to read. Let's move it to an inner function rather than lambda function

```
def init_client() -> SnykClient:
    def parse_list(value: str) -> list[str]:
        return [item.strip() for item in value.split(',')] if value else []

    return SnykClient(
        ocean.integration_config["token"],
        ocean.integration_config["api_url"],
        ocean.integration_config.get("app_host"),
        parse_list(ocean.integration_config.get("organization_id", "")),
        parse_list(ocean.integration_config.get("groups", "")),
        ocean.integration_config.get("webhook_secret"),
    )
```

Reply:
it has been seperated too